CONTRIBUTING.md

```diff
@@ -197,4 +197,21 @@ to merge**.
   - Send staging tag as PR for review.
 - Create a tag `v{version}` on the merged PR and push remote.
 
+## Merging existing ECS conventions
+
+The Elastic Common Schema (ECS) is being merged into OpenTelemetry Semantic
+Conventions per [OTEP 222][otep222]. When adding a semantic convention that
+exists in some form in ECS, consider the following guidelines:
+
+- Prefer using the existing ECS name when possible. In particular:
+  - Do not amend ECS names to follow plural guidelines or other soft
```

Comment:
I guess the goal of this sentence is to say something like: "Don't change the ECS names just to comply with OTel attribute guidelines" right? Like the pluralization rules.

In that case, what about we go with something more generic:


```suggestion
  - ECS names should not be modified to comply with OpenTelemetry general attribute guidelines. 
```

OR
```suggestion  
  - Special consideration must be applied for ECS attribute names that do not comply with the OpenTelemetry general attribute guidelines. 
```

OR

```suggestion
 - If an ECS attribute name does not comply with the OpenTelemetry general attribute guidelines and changing it is not desirable, provide data to support why changing the ECS attribute change cannot be done.
```

These are just suggestions.

Reply:
@joaopgrassi I like that direction!

How about?:

> ECS names should not be modified for the _only reason_ of complying with OpenTelemetry general attribute guidelines.